quarkus/dist/src/main/content/bin/kc.bat

```diff
@@ -177,8 +177,12 @@ if "x%JAVA%" == "x" (
 
 set CLASSPATH_OPTS="%DIRNAME%..\lib\quarkus-run.jar"
 
+rem set the homedir with \ replaced by /
+set KC_HOME_DIR=%DIRNAME%..
+set KC_HOME_DIR=%KC_HOME_DIR:\=/X%
```

Comment:
I think this would result to something like ` C:/XProgram Files/XKeycloak`.

The variant without `X` should work on windows:
 ```suggestion
set KC_HOME_DIR=%KC_HOME_DIR:\=/%
```

Here is a quick demonstration:
![image](https://github.com/user-attachments/assets/326e562d-4c3f-47ca-a471-bd6c9db6c02b)

Reply:
```suggestion
set KC_HOME_DIR=%KC_HOME_DIR:\=/%
```
@shawkins also, this somehow got back to the code, yet it should not be there as we discussed above. Any reason for reintroducing it or just a mistake?

It's also the reason why the CI build failed.